Configuration/Generator/python/VBFHZZ4Nu_14TeV_cfi.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         #pythiaHepMCVerbosity = cms.untracked.bool(False),
```

Comment:
Why is this commented out? I assume False is fine for your purposes no?

Reply:
Right, I'm happy to uncomment this line. (Sorry, it was just a copy-paste holdover from another config.)